### PR TITLE
Fix some IDE reported issues

### PR DESCRIPTION
**Pull request by yoksire**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
* Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by neeldoshii** [comment]
@yoksire What is this change about?

File: AnkiDroid/src/main/java/com/ichi2/libanki/Consts.kt [hunk 1]
```diff
@@ -93,7 +93,7 @@ object Consts {
     // The labels defined in consts.py are in AnkiDroid's resources files.
     const val DEFAULT_DECK_ID: Long = 1
 
-    val FIELD_SEPARATOR = Character.toString('\u001f')
+    val FIELD_SEPARATOR: String = Character.toString('\u001f')
```
Comment (neeldoshii) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Consts.kt:96]: Is this really needed?

Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Consts.kt:96]: might as well fix all the issues on the line while you're here

```suggestion
    const val FIELD_SEPARATOR: String = '\u001f'.toString()
```

**Review by david-allison — CHANGES_REQUESTED** [review]
Thanks for the commit! 

Could you fill out the commit template and rebase + push your change to provide it with a relevant commit message

**Pull request closed without merging**